tidy3d/components/eme/simulation.py

```diff
@@ -306,6 +307,7 @@ def plot_eme_ports(
         ax: Ax = None,
         hlim: Optional[tuple[float, float]] = None,
         vlim: Optional[tuple[float, float]] = None,
+        transpose: bool = False,
```

Comment:
Untested!  I could not find an example from tidy3d-notebooks which tests this function.

The changes to this function should be moved to a different PR.